src/nginx/module.cc

```diff
@@ -70,6 +70,8 @@ ApiManager::ApiBasepathRewriteAction ngx_esp_rewrite_uri(
   } else if (unparsed_uri_action ==
              ApiManager::ApiBasepathRewriteAction::REJECT) {
     return unparsed_uri_action;
+  } else {
```

Comment:
how about just one "else" block for line 70-75

Reply:
I got confused. If the unparsed_uri_action is ::NONE, we need to rewrite r->uri. 
So we should not return for ::NONE

I will revert the change.